latest/wfi_mosaic.yaml

```
@@ -13,49 +13,324 @@ type: object
+                enum: ["TAN"]
+                maxLength: 50
+                archive_catalog:
+                  datatype: nvarchar(50)
+                  destination: [WFIMosaic.projection]
+              s_region:
+                title: Spatial Extent of the Image
+                description: >
+                  The region of the sky enclosed by the pixel data
+                  contained within this file. This is given as a polygon stored as
+                  a string with the vertices defined by a list of right ascension
+                  and declination pairs in units of degrees.
+                type: string
+                archive_catalog:
+                  datatype: nvarchar(max)
+                  destination: [WFIMosaic.s_region]
+              image_shape:
+                title: Shape of Image (pixels)
+                description: Shape of the image (nx, ny) in units of pixels.
+                type: array
+                minItems: 2
+                maxItems: 2
+                items:
+                  type: integer
+                archive_catalog:
+                  datatype: nvarchar(50)
+                  destination: [WFIMosaic.image_shape]
+              ra:
+                title: Right Ascension of Image Center (deg)
+                description: Right ascension in units of degrees at the center of the image.
+                type: number
+                archive_catalog:
+                  datatype: float
+                  destination: [WFIMosaic.ra]
+              dec:
+                title: Declination of Image Center (deg)
+                description: Declination in units of degrees at the center of the image.
+                type: number
+                archive_catalog:
+                  datatype: float
+                  destination: [WFIMosaic.dec_center]
+              orientation:
+                title: Image Orientation (deg)
+                description: >
+                  Angle in units of degrees between celestial North and
+                  the image y-axis at the center of the image turning positive in
+                  the direction of increasing right ascension.
+                type: number
+                archive_catalog:
+                  datatype: float
+                  destination: [WFIMosaic.orientation]
+              orientation_ref:
+                title: Projection Region Orientation (deg)
+                description: >
+                  Angle in units of degrees between celestial North and
+                  the image y-axis at the center of the projection region turning
+                  positive in the direction of increasing right ascension. If this
+                  product does not use the Roman sky tessellation, then this
+                  keyword will be equal to meta.wcsinfo.orientation.
+                type: number
+                archive_catalog:
+                  datatype: float
+                  destination: [WFIMosaic.orientat]
```

This is also not updated?
```suggestion
                  destination: [WFIMosaic.orientation_ref]
```